kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/MetricsST.java

```diff
@@ -126,14 +135,17 @@ void beforeEach(String namespace) throws InterruptedException {
 
         String scraperPodName = kubeClient().listPodsByPrefixInName(namespace, scraperName).get(0).getMetadata().getName();
         LOGGER.atInfo().setMessage("Given Kroxylicious in {} namespace with {} replicas").addArgument(namespace).addArgument(1).log();
+        kroxyliciousOperator = new KroxyliciousOperator(Constants.KROXYLICIOUS_OPERATOR_NAMESPACE, 1);
+        kroxyliciousOperator.deploy();
         Kroxylicious kroxylicious = new Kroxylicious(namespace);
-        kroxylicious.deployPortPerBrokerPlainWithNoFilters(clusterName, 1);
-        bootstrap = kroxylicious.getBootstrap();
+        kroxylicious.deployPortIdentifiesNodeWithNoFilters();
+        bootstrap = kroxylicious.getBootstrap(clusterIpServiceName);
         kroxyliciousCollector = new MetricsCollector.Builder()
                 .withScraperPodName(scraperPodName)
+                .withOperatorNamespace(Constants.KROXYLICIOUS_OPERATOR_NAMESPACE)
                 .withNamespaceName(namespace)
                 .withComponentType(ComponentType.KROXYLICIOUS)
-                .withComponentName(Constants.KROXY_DEPLOYMENT_NAME)
+                .withComponentName(Constants.KROXYLICIOUS_OPERATOR_DEPLOYMENT_NAME)
```

Comment:
Both the Kroxylicious operator and operand (Kroxylicious Proxy) will emit metrics.     We'll need to system test both.  I would expect MetricsST to keep focusing on the operand's metrics.  In time there will be a new ST concerned with Operator metrics.

Reply:
You are right. I went in the wrong direction.

I have a question regarding this. Is there a plan to add selectors to the Kroxylicious Proxy deployment? Currently it only has labels and we are working here with selectors. I can refactor it to use labels, but I wonder if I can avoid to change this.

Reply:
Could you be specific with your question?

In general we should all be helping guide the project towards best practice and best solution.    So if you see a gap, like the Operator not doing something right, raise an issue stating the behaviour you see and the behavior you'd like and what use cases it supports. Metrics collecting is definitely an important use-case.

Reply:
Sorry, I was doing it wrong and I thought there was an issue there. I have already fixed.